src/react-components/ui-root.js

```diff
@@ -411,19 +411,23 @@ class UIRoot extends Component {
       if (hasAudio) {
         this.setState({ entryStep: ENTRY_STEPS.mic_granted });
       } else {
-        this.beginAudioSetup();
+        this.beginOrSkipAudioSetup();
       }
     } else {
-      this.beginAudioSetup();
+      this.beginOrSkipAudioSetup();
     }
   };
 
   onProfileFinished = () => {
     this.setState({ showProfileEntry: false });
   };
 
-  beginAudioSetup = () => {
-    this.setState({ entryStep: ENTRY_STEPS.audio_setup });
+  beginOrSkipAudioSetup = () => {
+    if (!this.props.forcedVREntryType || !this.props.forcedVREntryType.endsWith("_now")) {
+      this.setState({ entryStep: ENTRY_STEPS.audio_setup });
+    } else {
+      setTimeout(this.onAudioReadyButton, 3000); // Need to wait otherwise input doesn't work :/
```

Comment:
I think you were running into this setTimeout?

https://github.com/mozilla/hubs/blob/master/src/hub.js#L402

If you bypass that for `qs.vr_entry_type.endsWith("_now")`, you don't need this delay.

Reply:
I think i'll probably leave this PR as-is, since it seems better to keep the `_now` check in one place